test/integration/targets/bigip_device_license/tasks/issue-1810.yaml

```diff
@@ -0,0 +1,29 @@
+- import_tasks: setup.yaml
+
+- name: Try to license device
+  bigip_device_license:
+      license_key: "{{ license_key_to_use }}"
+      license_server: "{{ license_server }}"
+      accept_eula: yes
+      state: "latest"
+  register: result
+
+- name: Assert Try to license device
+  assert:
+      that:
+          - result is success
+          - result is changed
+
+- name: Try to license device  - Idempotent check
+  bigip_device_license:
+      license_key: "{{ license_key_to_use }}"
+      license_server: "{{ license_server }}"
+      accept_eula: yes
+      state: "latest"
+  register: result
+
+- name: Assert Try to license device  - Idempotent check
+  assert:
+      that:
+          - result is success
+          - result is not changed
```

Comment:
If this supposed to be not idempotent why this check?

Reply:
I have removed the Idempotent check. Essentially, I kept it there because the state is Idempotent when license is valid  and it would update/change only when license is no longer valid. 

BTW, I also included new "force_renewal" parameter intended to trigger renewal even when license is still valid. 

Let me know if you have any questions/concerns

Reply:
force is enough

Reply:
I have renamed the new parameter to "force"